src/loc/mystrings.d.ts

```diff
@@ -353,9 +355,10 @@ declare interface IDateTimeStrings {
   L_RelativeDateTime_XDaysFutureIntervals: string;
   L_RelativeDateTime_XDaysIntervals: string;
   L_RelativeDateTime_Today: string;
```

Comment:
Just for consistency, should this be moved to the top so that it's next to the other strings used by this control?